expression/src/dependencies.rs

```diff
@@ -31,7 +31,7 @@ impl ExpressionDependencies {
             .current_dir(cargo_workspace.path())
             .arg("build")
             .arg("--release")
-            .arg("--frozen")
+            .arg("--offline")
```

Comment:
why offline? We need to download them but not change the cargo lock

Reply:
> Prevents Cargo from accessing the network for any reason. Without this flag, Cargo will stop with an error if it needs to access the network and the network is not available. With this flag, Cargo will attempt to proceed without the network if possible.

only version resolution or download of data? the latter we cannot use

Reply:
[f6a6d1e](https://github.com/geo-engine/geoengine/pull/994/commits/f6a6d1e9ebd3804f6a9f0e62efe919c547bac161)